app/src/main/java/org/mozilla/fenix/home/recenttabs/view/RecentTabs.kt

```diff
@@ -118,6 +138,67 @@ private fun RecentTabItem(
     }
 }
 
+/**
+ * A recent search group item.
+ *
+ * @param searchTerm The search term for the group.
+ * @param tabId The id of the last accessed tab in the group.
+ * @param url The loaded URL of the last accessed tab in the group.
+ * @param icon The icon of the group.
+ * @param count Count of how many tabs belongs to the group.
+ * @param onSearchGroupClicked Invoked when the user clicks on a group.
+ */
+@Suppress("LongParameterList")
+@Composable
+private fun RecentSearchGroupItem(
```

Comment:
I did notice this. I think this could be a followup as we try to figure out what is common between the two and find a generic way composable function to move forward.